torchvision/io/_video_opt.py

```diff
@@ -1,10 +1,10 @@
 
-import importlib
+import importlib.machinery
```

Comment:
According to `mypy`, the second argument to `importlib.machinery.FileFinder` should be `Tuple[Loader, List[str]]` (see [CI fail](https://app.circleci.com/pipelines/github/pytorch/vision/3448/workflows/01102881-92e1-4d9e-addb-f6869f8d80f6/jobs/193510)). Thus it expects a `Loader` instance, but we are currently passing a class (see L18-23). Is this intentional? If so, I'll just add an ignore.

Reply:
Hum, interesting. `FileFinder` seems to support both [an instance (callable) and a class](https://docs.python.org/3/library/importlib.html#importlib.util.LazyLoader.factory).

I would not want to change this as of now without further extensive testing, so I would go for an ignore and a comment describing this for now.